automation/build-artifacts.sh

```diff
@@ -3,17 +3,18 @@
 ROOT_PATH=$PWD
 
 # remove any previous artifacts
-rm -rf ../ovirt-build ../rhv-build
+rm -rf ../ansible_collections
```

Comment:
if there are other ansible collections do we want to delete them too?

Reply:
this is just in CI automation which tests the collection
and when it reruns it has the previous build in it

Reply:
what I meant when I have a few other collections that already installed 
I think if we will remove the folder ansible_collections
we can remove the ones that we already installed but if we will be more specific like:
 rm -rf ../ansible_collections/ovirt  ../ansible_collections/redhat
we can be saved for both...wdyt?

Reply:
Yes, you are right but this change does not affect the build script. When you run the build script it will not remove the previous collections.
The automation folder is only for tests of the collection inside Jenkins.

Reply:
ok, thanks!!